libs/sdk-ui-pivot/src/CorePivotTable.tsx

```diff
@@ -352,6 +356,24 @@ export class CorePivotTableAgImpl extends React.Component<ICorePivotTableProps,
         );
     }
 
+    /**
+     * Checks DOM if height has been set on PivotTable parent wrapper. If height has not been set,
+     * PivotTable will not be rendered correctly. Therefore, we need to inform the user at least on the console
+     * on what is happening.
+     *
+     * @private
+     */
+    private alertParentWrapperMissingHeight(): void {
+        const parentWrapper = document.getElementById(this.pivotTableId)?.parentElement;
+        const parentHeight = parentWrapper?.offsetHeight || "0";
```

Comment:
Now that parentHeight is number, this should be `const parentHeight = parentWrapper?.offsetHeight ?? 0;`